### DISCUSSION TITLE
Selective Synchronization

### DISCUSSION THREAD

**Discussion by joowdx** [ChromaControl/ChromaControl#316] [General]
First off, this app is great! However all of my asus components are being synced up. Any ways to make it only sync those devices that are currently in sync with armoury crate? I want my mouse and keyboard to have separate effect not in sync with synapse/aura.

**Comment by andrewbabbittdev** [comment]
Historically this has never been possible with the AURA SDK, but ASUS could have changed that recently.

For right now as far as I'm aware, only Corsair supports something like this using ICUE.

**Reply by joowdx** [reply]
That's a complete bummer on ASUS' side. Their software freezes my mouse randomly when synchronizing with aura sync. That's why I prefer the onboard rgb setting.

**Reply by andrewbabbittdev** [reply]
Yeah, I myself may be moving more towards Gigabyte for things like my MB because ASUS quality has gone down so much over recent years. It's become very frustrating honestly.
